styleguide.config.js

```diff
@@ -21,24 +21,14 @@ module.exports = {
         {
             name: "Breadcrumbs",
             content: "packages/wonder-blocks-breadcrumbs/docs.md",
-            components: [
```

Comment:
So much moved over!

Reply:
Totally! this shows that we are pretty close to finish the migration 🤞